### DISCUSSION TITLE
Get result from bytes

### DISCUSSION THREAD

**Discussion by mainmaster** [ria-com/nomeroff-net#243] [Q&A] [answered]
How can I run a script to get result from bytes and not from path for file?

**Comment by dimabendera** [comment, marked as answer]
As an example
server
https://github.com/ria-com/nomeroff-net/blob/v3.2/examples/py/rest_examples/simple_fastapi/server.py#L41

client
https://github.com/ria-com/nomeroff-net/blob/v3.2/examples/py/rest_examples/simple_fastapi/test_detect_from_bytes.py
